lib/mix/tasks/openapi.spec.yaml.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Mix.Tasks.Openapi.Spec.Yaml do
+  @moduledoc """
+  Serialize the given OpenApi spec module to a YAML file.
+
+  ## Examples
+
+      $ mix openapi.spec.yaml --spec PhoenixAppWeb.ApiSpec apispec.yaml
+
+  ## Command line options
+   * `openapi.spec.json` - includes available options
+
+  """
+  use Mix.Task
+  require Mix.Generator
+
+  @default_filename "openapi.yaml"
+
+  @impl Mix.Task
+  def run(argv) do
+    Mix.Task.run("app.start")
```

Comment:
Ideally such tasks shouldn't start the whole app. This should be suitable to run in a CI stage where not all the app dependencies are available (think DB, Redis, ..). Also starting the app might have unwanted side-effects in some environments (sending a notification, attempting to retrieve data).

@mbuhot My understanding is that the original task might have been set to start the app for

```elixir
 %OpenApi{
      servers: [
        # Populate the Server info from a phoenix endpoint
        Server.from_endpoint(Endpoint) # 👈  This
      ]
}
```

for the server URLs to be set which depends on a Phoenix cache populated at runtime. If there's no other way to retrieve the URLs from Phoenix, we can only start the Endpoint supervisor.

Reply:
Could you help me with this one? I am new to elixir (

Reply:
I'll create a separate issue for this. It won't block this PR.